bond_test.go

```diff
@@ -51,10 +63,23 @@ type User struct {
 	Username  string `db:"username"`
 }
 
+func (u User) AfterCreate(ctx bond.Session) {
```

Comment:
This `AfterCreate` runs within the context passed by the caller. This context can be a normal session or a transaction. No need to keep track of a global here.